anatomy/track-tooling/analyzers

```diff
@@ -1 +0,0 @@
-This has moved to https://github.com/exercism/docs/tree/main/building/tooling/analyzers
```

Comment:
Please don't remove this file, as it is there to gracefully handle old links.

Reply:
Hello @ErikSchierboom ! I've revert this file removal (and the 2 others as well)